addons/website_event_meet/models/event_meeting_room.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import uuid
+from werkzeug.urls import url_join
+
+from odoo import api, fields, models
+from odoo.addons.http_routing.models.ir_http import slug
+
+
+class WebsiteEventMeetingRoom(models.Model):
+    _name = "event.meeting_room"
+    _description = "Event Meeting Room"
+    _order = "pinned DESC, last_joined DESC, id"
+
+    name = fields.Char("Topic", required=True, size=50)
+    active = fields.Boolean(default=True)
```

Comment:
shouldn't we track this field to know when it has been deactivated ? (and check the room lifetime ?)
Just an idea..

Reply:
@dbeguin So the meeting room will need to inherit from `mail.thread` right ?

Reply:
(not sure it's a relevant to have a chatter for this model)

Reply:
ha, pas faux ! Maybe not relevant indeed